apps/jumpserver/views/other.py

```diff
@@ -102,7 +102,7 @@ def versions_content(self):
         OPENSSH_VERSION=v9.4.0.0
         TINKER_VERSION=v0.1.6
         VIDEO_PLAYER_VERSION=0.2.0
-        CLIENT_VERSION=v3.0.0
+        CLIENT_VERSION=v3.0.1
         """
 
     def get_meta_json(self):
```

Comment:
I can't see the code that you're referring to because it was not provided as part of your inquiry. Please share the relevant code so I could assist with checking for errors, improvements, or inconsistencies within its structure and content!